docs/tutorials/working-with-llms.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Working with LLMs using Open WebUI
+---
+
+The **Rancher Desktop's Open WebUI extension** provides an easy-to-install setup, comprising the components listed below, for local GenAI use and development.
+
+- [Ollama](https://ollama.com/) to pull, run, fine-tune open-source LLMs
+- [Open WebUI](https://openwebui.com/) to chat with the LLMs, use custom knowledge, web search etc., via a feature rich Graphical User Interface (GUI)
+- A lightweight LLM called [tinyllama](https://ollama.com/library/tinyllama) to enable you kickstart your LLM exploration
+- [SearXNG](https://github.com/searxng/searxng) for web search to enhance retrieval augmented generation (RAG) results
+
+## Installing the Open WebUI extension
+
+:::important
+The Open WebUI extension depends on the host machine's ports 11434, 11500 and 11505 for the Ollama, Open WebUI, and the SearXNG services respectively. Please ensure other applications or services on your machine do not occupy these host ports.
+:::
+
+### Minimum Requirements
+
+In addition to the general Rancher Desktop [installation requirements](../getting-started/installation.md), you should meet the requirements listed below for the Open WebUI extension. 
+
+- Rancher Desktop version 1.17 or above
+- 7 GB available disk space
+- 16 GB RAM (recommended)
+
+### Steps to install
+
+You can install the Open WebUI extension via the [Extensions Catalog](../ui/extensions.md#catalog) GUI or via the [rdctl CLI](../references/rdctl-command-reference.md#rdctl-extension-install).
```

Comment:
I just added it as a way for those advanced users who may have scripted setups to install, launch. uninstall RD and they may want to install some extensions via the script as well. 

> It encourages users to install the latest tag, ..
I think we can add a note here recommending people to use release tags instead of `latest`